pageserver/src/controller_upcall_client.rs

```diff
@@ -173,6 +174,7 @@ impl ControlPlaneGenerationsApi for ControllerUpcallClient {
                         listen_pg_port: m.postgres_port,
                         listen_http_addr: m.http_host,
                         listen_http_port: m.http_port,
+                        use_https: m.use_https,
```

Comment:
Just making a note that afaict there is no cross-version compatibility issue here, because `NodeRegisterRequest` ignores unknown fields during deserialization.

Reply:
Yes, unknown fields are ignored, everything should be compatible here.
I also added `#[serde(default)]` to `TenantLocateResponseShard`/`NodeDescribeResponse` so clients won't fail if they get the response from an old storage controller instance without the `use_https` field.

Reply:
Good catch, thanks.